apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts

```diff
@@ -1506,4 +1507,16 @@ export class InvoicePreviewComponent implements OnInit, OnChanges, OnDestroy {
         });
         this.closeAdvanceReceiptModal();
     }
+
+    /**
+     * To open advance receipt adjust modal requested from invoice detailed component
+     *
+     * @param {*} event emmiter event
+     * @memberof InvoicePreviewComponent
+     */
+    public openAdvanceReceiptModal(event): void {
+        if (event) {
```

Comment:
Unused variable event

Reply:
It's get calling when component render and pop up get openning without event so event is passing here